src/Server/Local.php

```diff
@@ -50,19 +50,20 @@ public function connect()
     /**
      * {@inheritdoc}
      */
-    public function run($command)
+    public function run($command, array $env = [])
```

Comment:
I don't like adding `$env` to run function. Why not just pass env as `ENV=val composer ...`?